docs/intro.rst

```diff
@@ -72,7 +72,7 @@ In addition to the core functionality, pybind11 provides some extra goodies:
 
 - Binaries are generally smaller by a factor of at least 2 compared to
   equivalent bindings generated by Boost.Python. A recent pybind11 conversion
-  of `PyRosetta`_, an enourmous Boot.Python binding project, reported a binary
+  of `PyRosetta`_, an enourmous Boost.Python binding project, reported a binary
```

Comment:
While you're at it, the previous word is also misspelled: enourmous -> enormous.